public/js/utils.jsx

```diff
@@ -87,3 +85,19 @@ export function parseQuery(url){
 
   return data;
 }
+
+
+/**
+ * Given a list of pay-method objects return one that matches
+ * the payMethodUri. If no matches returns null.
```

Comment:
last sentence could use some re-wording, maybe : 'If none match, it returns null'